tests/test_download.py

```diff
@@ -97,30 +99,32 @@ def test_download_configs(self):
     #
     # Tests for 'wf_use_local_configs'
     #
-    def test_wf_use_local_configs(self):
+    @with_temporary_folder
+    def test_wf_use_local_configs(self, tmp_path):
         # Get a workflow and configs
-        test_pipeline_dir = os.path.join(tempfile.mkdtemp(), "nf-core-testpipeline")
+        test_pipeline_dir = os.path.join(tmp_path, "nf-core-testpipeline")
         create_obj = nf_core.create.PipelineCreate(
             "testpipeline", "This is a test pipeline", "Test McTestFace", outdir=test_pipeline_dir
         )
         create_obj.init_pipeline()
 
-        test_outdir = tempfile.mkdtemp()
-        download_obj = DownloadWorkflow(pipeline="dummy", revision="1.2.0", outdir=test_outdir)
-        shutil.copytree(test_pipeline_dir, os.path.join(test_outdir, "workflow"))
-        download_obj.download_configs()
+        with tempfile.TemporaryDirectory() as test_outdir:
```

Comment:
In this test I need two temporary directories. The first come from decorator. The second with *context manager*. Code is the same, but it seems to be a lot of changes